erpnext/patches/v4_0/reset_permissions_for_masters.py

```diff
@@ -16,5 +16,5 @@ def execute():
 		try:
 			reset_perms(doctype)
 		except:
-			print "Error resetting perms for", doctype
+			print("Error resetting perms for", doctype)
 			raise
```

Comment:
Um. I read accepted answer from [this Stack Overflow question](https://stackoverflow.com/q/2052390). I'm confused. 

Anyways. That doesn't really prevent me from running `bench --site site1.local install-app erpnext` as of now. I'll tackle that when it becomes a problem. Cool?